3-templ/README.md

```diff
@@ -14,27 +14,60 @@ To add templ to your `go.mod` file, run the following command:
 go mod tidy
 ```
 
+For a better dev experience, we recommend installing the official `templ-vscode` plugin via the VS Code Marketplace for syntax highlighting.
+
 ## Using templ in this codebase
 
-templ components in files with the `.templ` extension can be treated and edited as normal HTML. Once you are happy with your changes, run the following command to generate Go templates for rendering HTML from your `.templ` files:
+templ components in files with the `.templ` extension can be treated and edited as normal HTML. Once you are happy with your changes, run the following command to generate Go templates for rendering HTML from your `.templ` files.
```

Comment:
It's not quite normal HTML. It has templ keywords in it. It's actually different from most templating languages which the default go templates would be. 

I'd say it's templ functions with plain html in them